src/validations/identityDocumentsGroup2.ts

```diff
@@ -0,0 +1,6 @@
+import { body } from "express-validator";
+
+export const identityDocsGroup2Validator = [
+    body("documentsGroup2A", "identityDocumentsGroup1Empty").notEmpty(),
+    body("documentsGroup2B", "identityDocumentsGroup1Empty").notEmpty()
```

Comment:
User needs to select 2 documents but with at least 1 from group A but can choose 2 from group A and 0 from group B